actions/watch.ts

```diff
@@ -14,7 +14,7 @@ export const checkForAndPlaceOrder: ActionFn = async (
   const registry = await Registry.load(context, blockEvent.network);
   const chainContext = await ChainContext.create(context, blockEvent.network);
 
-  const contractsToDelete = []
+  const contractsToDelete: string[] = []
```

Comment:
Where are we deleting the contracts addresses?

It's seems we never add to this array.

Reply:
Better to see if everything works fine before we start deleting contracts again as last time deleted unwanted contracts.

Reply:
Yeah, but leaving it here doesn't make sense then. We should add it when we add the functionality.